packaging/fedora/snapd.spec

```diff
@@ -320,11 +331,22 @@ export GOPATH=$(pwd):%{gopath}
 export GOPATH=$(pwd):$(pwd)/Godeps/_workspace:%{gopath}
 %endif
 
-%gobuild -o bin/snap %{import_path}/cmd/snap
-%gobuild -o bin/snap-exec %{import_path}/cmd/snap-exec
-%gobuild -o bin/snapctl %{import_path}/cmd/snapctl
-%gobuild -o bin/snapd %{import_path}/cmd/snapd
-%gobuild -o bin/snap-update-ns %{import_path}/cmd/snap-update-ns
+# We have to build snapd first to prevent the build from
+# building various things from the tree without additional
+# set tags.
+GOFLAGS=
+%if 0%{?with_test_keys}
+GOFLAGS="-tags withtestkeys"
+%endif
+
+# We have to build snapd first to prevent the build from
```

Comment:
I don't think anything else cares about test keys, if that lets you simplify things, go for it!